### DISCUSSION TITLE
I do not own a cellphone and never will, how can I have 2FA?

### DISCUSSION THREAD

**Discussion by RobotMasterLord** [community/community#82801] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I haven't found a way to activate two factor authentification without a phone.
If I can't do this, I'll let my account die, so be it.

Anyone found a way to activate 2FA without a phone?

**Comment by ananuness** [comment, marked as answer]
Hii! You can use any open source password manager that generates 2FA codes as KeepassXC or others. About keepass, you can see more [here](https://stackoverflow.com/questions/68824508/is-there-a-free-way-to-set-up-github-2fa-without-a-mobile-device), and about other methods you can check this [thread on reddit](https://www.reddit.com/r/github/comments/15z9y1r/is_there_a_way_to_use_github_2fa_without_using_a/)

**Reply by RobotMasterLord** [reply]
Thank you, that's a very useful answer. I'll try that out.

**Reply by ananuness** [reply]
I hope it helps!!

**Comment by ghostinhershell** [comment]
Hi @RobotMasterLord,

Like @ananuness mentioned above, if you can’t or won’t use a mobile device for 2FA, there are multiple options for standalone TOTP applications that run across platforms. https://keepassxc.org/ was recommended by community member @ldez, which is an open, free desktop application, and for browser-based plugins there's [1Password](https://support.1password.com/one-time-passwords/#to-save-your-qr-code-using-the-1password-browser-extension). Any code that is compatible with RFC 6238 will work, using the manual setup options documented in "[Configuring two-factor authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-a-totp-mobile-app)".
